storm-core/src/jvm/org/apache/storm/command/CLI.java

```diff
@@ -238,10 +238,13 @@ public CLIBuilder arg(String name, Parse parse, Assoc assoc) {
             DefaultParser parser = new DefaultParser();
             CommandLine cl = parser.parse(options, rawArgs);
             HashMap<String, Object> ret = new HashMap<>();
-            for (Opt opt: opts) {
+            for (Opt opt : opts) {
                 Object current = null;
-                for (String val: cl.getOptionValues(opt.shortName)) {
-                    current = opt.process(current, val);
+                String[] strings = cl.getOptionValues(opt.shortName);
+                if (strings != null) {
+                    for (String val : cl.getOptionValues(opt.shortName)) {
```

Comment:
replace `cl.getOptionValues(opt.shortName)` with `strings` to avoid duplicate getOptionValues